core/domain/rights_manager.py

```diff
@@ -256,6 +256,65 @@ def get_multiple_exploration_rights_by_ids(exp_ids):
     return exp_models_list
 
 
+def _get_activity_rights_where_user_is_owner(activity_type, user_id):
```

Comment:
This and the next two methods have "owner" in the name but "some role" in the docstring, which is vague (particularly because for some entities "role" can just be "viewer").

If you mean the latter, change the method name to say "where_user_has_named_role" perhaps, or "where_user_has_role" if the former name is too long.

Reply:
Done.